server/src/services/boards/sessions.go

```diff
@@ -244,12 +271,23 @@ func (s *BoardSessionService) CreatedSession(board uuid.UUID, session database.B
 }
 
 func (s *BoardSessionService) UpdatedSession(board uuid.UUID, session database.BoardSession) {
-	err := s.realtime.BroadcastToBoard(board, realtime.BoardEvent{
-		Type: realtime.BoardEventParticipantUpdated,
-		Data: new(dto.BoardSession).From(session),
-	})
+	connectedBoards, err := s.database.GetSingleUserConnectedBoards(session.User)
 	if err != nil {
-		logger.Get().Errorw("unable to broadcast updated board session", "err", err)
+		return
+	}
+	for _, session := range connectedBoards {
+		userSession, err := s.database.GetBoardSession(session.Board, session.User)
+		if err != nil {
+			logger.Get().Errorw("unable to get board session of user", "board", session.Board, "user", session.User, "err", err)
+			return
+		}
+		err = s.realtime.BroadcastToBoard(session.Board, realtime.BoardEvent{
+			Type: realtime.BoardEventParticipantUpdated,
+			Data: new(dto.BoardSession).From(userSession),
+		})
+		if err != nil {
+			logger.Get().Errorw("unable to broadcast updated board session", "err", err)
+		}
```

Comment:
I don't think it is necessary to update all active board sessions of user when the status changes on one board

Reply:
See above 😄 

Maybe we should investigate splitting the participant part (board session) and user part (account) going forward for clearer communication and less overhead per change.

Reply:
I think splitting would be a good idea

Reply:
Sounds good to me. 
I will open an issue for it since doing the splitting inside this PR would lead to more confusion than anything else.
To ensure this issue is resolved, I will work on it next before starting anything else.

Reply:
Thank you @BenedictHomuth

Reply:
Details here: #4217